Makefile

```diff
@@ -58,12 +58,28 @@ lint:
 man:
 	man/md2man-all.sh
 
-runcimage:
+runcimage: | .busybox .hello-world .debian
 	$(CONTAINER_ENGINE) build $(CONTAINER_ENGINE_BUILD_FLAGS) -t $(RUNC_IMAGE) .
 
+.hello-world:
+	$(CONTAINER_ENGINE) export $(shell $(CONTAINER_ENGINE) create hello-world) -o $(CURDIR)/tests/integration/testdata/hello-world.tar
+	chmod a+r $(CURDIR)/tests/integration/testdata/hello-world.tar
+	touch "$@"
+
+.busybox:
+	$(CONTAINER_ENGINE) export $(shell $(CONTAINER_ENGINE) create busybox) -o $(CURDIR)/tests/integration/testdata/busybox.tar
+	chmod a+r $(CURDIR)/tests/integration/testdata/busybox.tar
+	touch "$@"
+
+.debian:
+	$(CONTAINER_ENGINE) export $(shell $(CONTAINER_ENGINE) create debian) -o $(CURDIR)/tests/integration/testdata/debian.tar
+	chmod a+r $(CURDIR)/tests/integration/testdata/debian.tar
+	touch "$@"
+
 test: unittest integration rootlessintegration
 
-localtest: localunittest localintegration localrootlessintegration
+localtest: | all .busybox .hello-world .debian
```

Comment:
This repetition can be avoided by having something like
```make
TEST_IMAGES ?= debian busybox hello-world
GET_IMAGE_TARGETS := $(addprefix ., $(TEST_IMAGES))
...
localtest: | $(GET_IMAGE_TARGETS)
```

Reply:
Also, specifying these prerequisites here is redundant, as every one of (localunittest, localintegration, localrootlessintegration) already have those.